app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -27,6 +29,7 @@ data class ValidatedIntentData(val url: String, val source: Session.Source)
  */
 object IntentValidator {
     @VisibleForTesting const val DIAL_PARAMS_KEY = "com.amazon.extra.DIAL_PARAM"
+    private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
```

Comment:
nit: I prefer to put private constants at the top level, to reduce the amount of code in the class body, which makes it clearer what is important in the class.